climada_petals/hazard/wildfire.py

```diff
@@ -383,22 +403,29 @@ def set_proba_fire_seasons(self, n_fire_seasons=1, n_ignitions=None,
             ign_max = n_ignitions[1]
 
         prob_fire_seasons = [] # list to save probabilistic fire seasons
+        event_proba_new = [] #added by Sam. G.
         # create probabilistic fire seasons
         for i in range(n_fire_seasons):
+            self.ProbaParams.prop_proba = float(np.random.normal(self.ProbaParams.prop_proba_mean,
```

Comment:
You are here using the `np.random`, yet you also import `random`. Any good reason for that?